app/src/main/res/layout/fragment_next_up.xml

```diff
@@ -0,0 +1,76 @@
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:background="@drawable/moviebg">
+
+    <!-- Overlay -->
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/content"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@drawable/noise"
+        android:orientation="horizontal"
+        android:padding="16dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent">
+
+        <ImageView
+            android:id="@+id/image"
+            style="@style/Widget.Leanback"
+            android:layout_width="wrap_content"
+            android:layout_height="175dp"
+            android:adjustViewBounds="true"
+            android:maxWidth="400dp"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_goneMarginEnd="0dp"
+            tools:src="@drawable/app_logo" />
+
+        <TextView
+            android:id="@+id/nextup"
+            style="@style/TextAppearance.Leanback.DetailsDescriptionTitle"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            android:text="@string/lbl_next_up"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@id/image"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/title"
+            style="@style/TextAppearance.Leanback.DetailsDescriptionSubtitle"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@id/image"
+            app:layout_constraintTop_toBottomOf="@id/nextup"
+            tools:text="Episode title" />
+
+        <TextView
+            android:id="@+id/description"
+            style="@style/TextAppearance.Leanback.DetailsDescriptionBody"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:layout_marginStart="16dp"
+            android:ellipsize="end"
+            android:maxLines="6"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@id/image"
+            app:layout_constraintTop_toBottomOf="@id/title"
+            tools:text="Episode description" />
+    </androidx.constraintlayout.widget.ConstraintLayout>
+
+    <org.jellyfin.androidtv.playback.nextup.NextUpButtons
```

Comment:
Would move this into the Overlay layout

Reply:
This should be done at a later moment when we're supporting a negative timeout for the buttons. For now it's embedded in the fragment so the code is entirely separated.